src/rust/plugin-registry/src/static_files/hax_docker_analyzer.nomad

```diff
@@ -0,0 +1,284 @@
+      mode = "bridge"
+      // TODO i think? possibly cargo culted?
+      // dns {
+      //   servers = local.dns_servers
+      // }
+      port "plugin-execution-sidecar" {}
+      port "graph-query-proxy" {}
+    }
+
+    service {
+      name = "plugin-execution-sidecar-${var.plugin_id}"
+      tags = [
+        "plugin-execution-sidecar",
+        "tenant-${var.tenant_id}",
+        "plugin-${var.plugin_id}"
+      ]
+
+      connect {
+        sidecar_service {
+          proxy {
+            upstreams {
+              destination_name = "plugin-work-queue"
+              # port unique but arbitrary - https://github.com/hashicorp/nomad/issues/7135
+              local_bind_port = 1000
+            }
+
+            upstreams {
+              destination_name = "plugin-${var.plugin_id}"
+              # port unique but arbitrary - https://github.com/hashicorp/nomad/issues/7135
+              local_bind_port = 1001
+            }
+
+            upstreams {
+              destination_name = "graph-query"
+              # port unique but arbitrary - https://github.com/hashicorp/nomad/issues/7135
+              local_bind_port = 1002
+            }
+          }
+        }
+      }
+    }
+
+    # The execution task pulls messages from the plugin-work-queue and
+    # sends them to the plugin
+    task "plugin-execution-sidecar" {
+      driver = "docker"
+
+      config {
+        image = var.plugin_execution_sidecar_image
+      }
+
+      template {
+        data        = var.observability_env_vars
+        destination = "observability.env"
+        env         = true
+      }
+
+      env {
+        PLUGIN_EXECUTOR_PLUGIN_ID = var.plugin_id
+
+        // FYI: the upstream plugin's address is discovered at runtime, not
+        // env{}, because the upstream's name is based on ${PLUGIN_ID}.
+
+        PLUGIN_WORK_QUEUE_CLIENT_ADDRESS = "http://${NOMAD_UPSTREAM_ADDR_plugin-work-queue}"
+
+        RUST_LOG       = var.rust_log
+        RUST_BACKTRACE = 1
+      }
+    }
+
+    # A level of indirection in front of the Graph Query Service that prevents
+    # the plugin from querying other tenants' data.
+
+    task "graph-query-proxy-sidecar" {
+      driver = "docker"
+
+      config {
+        image = var.graph_query_proxy_image
+      }
+
+      template {
+        data        = var.observability_env_vars
+        destination = "observability.env"
+        env         = true
+      }
+
+      env {
+        GRAPH_QUERY_PROXY_BIND_ADDRESS = "0.0.0.0:${NOMAD_PORT_graph-query-proxy}"
+        GRAPH_QUERY_CLIENT_ADDRESS     = "http://${NOMAD_UPSTREAM_ADDR_graph-query}"
+        TENANT_ID                      = var.tenant_id
+
+        RUST_LOG       = var.rust_log
+        RUST_BACKTRACE = 1
+      }
+
+      resources {
+        cpu    = 25  // MHz
+        memory = 128 // MB
+      }
+    }
+
+    restart {
+      attempts = 1
+      delay    = "5s"
+    }
+  }
+
+  group "plugin" {
+    network {
+      mode = "bridge"
+      // TODO
```

Comment:
similar question, is this TODO something that needs to be done right now? If so let's do it, if not let's remove it

Reply:
it's not, but again, pre-existing